mesa/visualization/templates/js/ChartModule.js

```diff
@@ -80,12 +83,10 @@ var ChartModule = function(series, canvas_width, canvas_height) {
     };
 
     this.reset = function() {
-        chart.destroy();
-        chartData.labels = [];
-        chart = new Chart(context, {
-            type: 'line',
-            data: chartData,
-            options: chartOptions
+        while (chart.data.labels.length) { chart.data.labels.pop(); }
+        chart.data.datasets.forEach((dataset) => {
```

Comment:
While I think it is by and large appropriate to use `=>` notation for declaring functions, since we are not using any sort of build process, it is best for the time to use the long-form declaration of functions.
```
chart.data.datasets.forEach(function(dataset) {
    while (dataset.data.length) {
        dataset.data.pop();
    }
})